server/models/comments.model.js

```diff
@@ -7,6 +7,7 @@ module.exports = function (app) {
   const comments = new mongooseClient.Schema({
     userId: { type: String, required: true, index: true },
     contributionId: { type: String, required: true, index: true },
+    parentCommentId: { type: String, default: '' },
     content: { type: String, required: true },
```

Comment:
We decided to have only one level of nesting: Contribution >> Comment >> Reply. Instead of a self reference, we could also create a new model `replies`. What is your opinion on it, @appinteractive ?

Reply:
done with `parentcommentId` and a comment.hook for getting the children per comment